lib/PriceSettings.js

```diff
@@ -33,54 +37,80 @@ class PriceSettings extends Component {
   }
 
   render() {
-    const { price, updateFeed, updateFiat, refreshPrice, chain } = this.props;
+    const {
+      price,
+      updateFeed,
+      updateFiat,
+      refreshPrice,
+      hideWidgetMenus,
+      chain,
+      priceWidget: { hideMenus },
+    } = this.props;
     const formattedChain = chain === 'bitcoincash' ? 'Bitcoin Cash' : chain;
 
     return (
-      <div className={'col'}>
-        <Header type="h3">Select your price settings</Header>
-        <Text type="p">
-          These settings will update the price display across your app. The
-          cryptocurrency is based on the node you are currently connected to (
-          <span style={{ textTransform: 'capitalize' }}>
-            {formattedChain || '...'}
-          </span>
-          ).
-        </Text>
-        <Text>
-          {price.crypto +
-            ' ' +
-            price.fiatSymbol +
-            this.moneyNumber(price.price) +
-            ' '}
-        </Text>
-        <select
-          value={price.fiat}
-          onChange={e => {
-            updateFiat(e.target.value);
-            refreshPrice();
-          }}
-        >
-          {price.availableFiats.map(val => (
-            <option value={val} key={val}>
-              {val}
-            </option>
-          ))}
-        </select>
-        &nbsp;
-        <select
-          value={price.feed}
-          onChange={e => {
-            updateFeed(e.target.value);
-            refreshPrice();
-          }}
-        >
-          {price.availableFeeds.map(val => (
-            <option value={val} key={val}>
-              {val}
-            </option>
-          ))}
-        </select>
+      <div className="col">
+        <div className="row mb-3">
+          <Header type="h3">Select your price settings</Header>
+          <Text type="p">
+            These settings will update the price display across your app. The
+            cryptocurrency is based on the node you are currently connected to (
+            <span style={{ textTransform: 'capitalize' }}>
+              {formattedChain || '...'}
+            </span>
+            ).
+          </Text>
+          <Text>
+            {price.crypto +
+              ' ' +
+              price.fiatSymbol +
+              this.moneyNumber(price.price) +
+              ' '}
```

Comment:
Dunno why this space doesn't look right in settings, it looks good in the footer?
I tried replacing this with a `&nbsp;` and got that little margin back between the elements

Reply:
Stupid JSX